src/components/ItineraryCalendar.tsx

```
@@ -0,0 +1,140 @@
+import React, { useState } from 'react';
+import { Calendar, dateFnsLocalizer } from 'react-big-calendar';
+import withDragAndDrop from 'react-big-calendar/lib/addons/dragAndDrop';
+import { format, parse, startOfWeek, getDay } from 'date-fns';
```

ESLint

[error] 'date-fns' import is restricted from being used. Please import date-fns functions individually from subpaths, e.g. `import format from 'date-fns/format'` instead of destructured imports.

(no-restricted-imports)